release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -85,11 +85,134 @@ The {product-title} installation program for Google Cloud Platform (GCP) now cre
+[id="ocp-4-8-ibm-z"]
+==== IBM Z and LinuxONE
+
+With this release, IBM Z and LinuxONE are now compatible with {product-title} {product-version}. The installation can be performed with z/VM or {op-system-base} KVM. For installation instructions see:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network]
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network]
+
+[discrete]
+===== Notable Enhancements
+
+The following new features are supported on IBM Z and LinuxONE with {product-title} {product-version}:
+
+* KVM on {op-system-base} 8.3 or later is supported as a hypervisor for user-provisioned installation of {product-title} {product-version} on IBM Z and LinuxONE. Installation with static IP addresses as well as installation in a restricted network are now also supported.
+* Encrypting data stored in etcd
+* 4K FCP block device
+* 3-node cluster support
+
+[discrete]
+===== Supported features
+
+The following features are also supported on IBM Z and LinuxONE:
+
+* Multipathing
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* OVN-Kubernetes with an initial installation of {product-title} 4.8
+* z/VM Emulated FBA devices on SCSI disks
+
+These features are available only for {product-title} on IBM Z for {product-version}:
+
+* HyperPAV enabled on IBM System Z /LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Precision Time Protocol (PTP) hardware
+
+* The following {product-title} features are unsupported:
+** Automatic repair of damaged machines with machine health checking
+** CodeReady Containers (CRC)
+** Controlling overcommit and managing container density on nodes
+** CSI volume cloning
+** CSI volume snapshots
+** FIPS cryptography
+** Helm command-line interface (CLI) tool
+** Multus CNI plug-in
+** NVMe
+** OpenShift Metering
+** OpenShift Virtualization
+** Tang mode disk encryption during {product-title} deployment
+
+* Worker nodes must run {op-system-first}.
```

Comment:
I think since there are currently only two list items that are technically complete sentences with periods at the end, and they're pretty short, it would look better to just drop the periods from those two, and sync everything up to just be punctuationless for this list (since it's otherwise a pretty long list of fragments overall).